subjects/modify_letter/README.md

```diff
@@ -0,0 +1,50 @@
+## modify_letter
+
+### Instructions
+
+Create a **function** `remove_letter_sensitive` that returns a string without the letter specified as argument
+
+Create a **function** `remove_letter_insensitive` that returns a string without the letter specified as argument (ignoring case)
+
+Create a **function** `swap_letter_case` that returns a string swapping the case for the chosen letter.
+
+If there is an empty character in the arguments, the function should return the string as being passed.
+
+If there is an empty string, it should return an empty string.
+
+### Expected Functions
+
+```rust
+pub fn remove_letter_sensitive(s: &str, letter: char) -> String {
+}
+
+pub fn remove_letter_insensitive(s: &str, letter: char) -> String {
+}
+
+pub fn swap_letter_case(s: &str, letter: char) -> String {
+}
+```
+
+### Usage
+
+Here is a program to test your functions.
+
+```rust
+use modify_letter::*modify_letter*;
+
+fn main() {
+    println!("{}", remove_letter_sensitive("Joje jis mijssjing", 'j'));
```

Comment:
Maybe it can be nice to modify the strings in the example so they won't be the same of the exercise the students already did in the data quest.

Reply:
Thanks, I changed them.